### DISCUSSION TITLE
Is there any possibility to reduce size of Screenshots ?

### DISCUSSION THREAD

**Discussion by amadeuszpolak** [pointfreeco/swift-snapshot-testing#706] [Q&A]
I would like to make smaller screenshots by default (1/4*currentScale) is there any way to do that? In current implementation 2MB for SS is way to much for my project.

**Comment by Sherlouk** [comment]
It's worth noting there is a HEIC plugin documented in the README. This has reduced the file size heavily for me! YMMV

**Reply by amadeuszpolak** [reply]
I did some reserach and overriding `displayScale` in `UITraitCollection` passing to the `ViewImageConfig` is working, obviously it will change the quality of the image because it is passing a lower scale instead of retriving it from the testing device.
